enclone_print/src/print_clonotypes.rs

```diff
@@ -105,11 +106,11 @@ pub fn print_clonotypes(
     // Define parseable output columns.  The entire machinery for parseable output is controlled
     // by macros that begin with "speak".
 
-    let mut max_chains = 4;
+    let max_chains = 4;
     // This seems like a bug, since rsi is uninitialized upon entry to print_clonotypes.
-    for r in rsi.iter() {
-        max_chains = max(max_chains, r.mat.len());
-    }
```

Comment:
Why is this commented out?

Reply:
Since this was an outstanding bug, I decided to leave the existing bugged code commented-out and the prior comment in case we are debugging something in the future. We can come back and trim it out later.